types/mobx-task/index.d.ts

```diff
@@ -45,7 +45,8 @@ export interface TaskStatusAware<Result = any, Args extends any[] = any[]> exten
 export interface TaskCreator<K extends keyof TaskOptions<any>> extends MethodDecorator, PropertyDecorator {
     <R, A extends any[]>(func: TaskFunc<R, A>, options?: Pick<TaskOptions<WithoutPromise<R>>, K>): TaskStatusAware<WithoutPromise<R>, A>;
 
-    (options: Pick<TaskOptions<WithoutPromise<any>>, K>): PropertyDecorator | MethodDecorator;
+    (options: Pick<TaskOptions<WithoutPromise<any>>, K>): MethodDecorator;
```

Comment:
I don't think there's a difference in the declaration of this type. Actually using the union type is the recommended practice.

What where you trying to fix?

Reply:
@armanio123 please check latest build errors. I forgot to add a test before. now i reverted change and added test to show you this errors